doc/user_guide/advanced.rst

```diff
@@ -197,3 +197,83 @@ callback (default to 1).
 
 .. |SlurmExecutor| replace:: ``submitit.SlurmExecutor``
 .. _SlurmExecutor: https://github.com/facebookincubator/submitit/blob/main/submitit/slurm/slurm.py#L214
+
+
+Benchopt configuration
```

Comment:
I would put this in the `benchmark workflow` as a separate file, maybe at the end, on the configuration of benchopt.